src/index.html

```diff
@@ -4,9 +4,336 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__top">
+        <div class="header__logo">
+          <a href="https://www.moyo.ua/">
+            <img src="./images/logo.png" alt="logo">
+          </a>
+        </div>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a class="nav__link is-active" href="#">Apple</a>
+            </li>
+  
+            <li class="nav__item">
+              <a class="nav__link" href="#">Samsung</a>
+            </li>
+  
+            <li class="nav__item">
+              <a class="nav__link" href="#">Smartphones</a>
+            </li>
+  
+            <li class="nav__item">
+              <a 
+                class="nav__link" 
+                href="#"
+                data-qa="nav-hover"
+              >
+                Laptops & computers
+              </a>
+            </li>
+  
+            <li class="nav__item">
+              <a class="nav__link" href="#">Gadgets</a>
+            </li>
+  
+            <li class="nav__item">
+              <a class="nav__link" href="#">Tablets</a>
+            </li>
+  
+            <li class="nav__item">
+              <a class="nav__link" href="#">Photo</a>
+            </li>
+  
+            <li class="nav__item">
+              <a class="nav__link" href="#">Video</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <div class="main">
```

Comment:
```suggestion
    <main class="main">
```

that's the reason of 'lemur')) tests look for main tag. use sematic tags